### PR TITLE
Pin the versions of the actions

### PR DESCRIPTION
**Pull request by smorimoto**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -24,10 +24,10 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@master
+        uses: actions/checkout@v2
 
       - name: Use OCaml ${{ matrix.ocaml-version }}
-        uses: avsm/setup-ocaml@v1.0.1
+        uses: avsm/setup-ocaml@v1
```
Comment (hcarty) [inline .github/workflows/main.yml:30]: Why the revert to v1?

**Review by hcarty — COMMENTED** [review]
Thank you for this! Just a quick version question before I merge.

File: .github/workflows/main.yml (hunk 1 above)
Comment (smorimoto) [reply .github/workflows/main.yml:30]: I really don't like the format that the GitHub team popularized, but it's explained here. (I added that recently.)
https://github.com/avsm/setup-ocaml#how-to-specify-the-version

Comment (hcarty) [reply .github/workflows/main.yml:30]: Ah ok, thank you for the reference!

**Pull request merged**